app/models/chess_piece.rb

```diff
@@ -84,4 +84,14 @@ def is_obstructed?(endpoint_x, endpoint_y)
     end
   end
 
+  def capture(endpoint_x, endpoint_y)
+    opponent_piece = game.piece_in_square?(endpoint_x, endpoint_y)
+
+    if self.color != opponent_piece.color
```

Comment:
well we're `in` the `chess_piece` in question, so i think this line is fine the way it is.